src/cell.rs

```diff
@@ -41,8 +41,8 @@ impl Cell {
 
     /// Create a new `Cell` initialized with content from `string`.
     /// By default, content is align to `LEFT`
-    pub fn new(string: &str) -> Cell {
-        Cell::new_align(string, Alignment::LEFT)
+    pub fn new<S: ToString>(string: S) -> Cell {
```

Comment:
Because `AsRef<str>` isn't implemented for the integers, for example, so I can't do:

`Cell::new(8)`

Like I gave in examples